### ISSUE TITLE
Basic init

### ISSUE THREAD

**Issue by killing333** [xmartlabs/XLPagerTabStrip#23]
Can you add support for basic init in case people are not using nib or storyboard?

**Comment by jimcn** [comment]
yeah

**Comment by MobileMon** [comment]
+1

**Comment by michaelEllisUy** [comment]
Hey guys,

I'm not sure what you mean by supporting basic init. 
The init method automatically calls the designated initializer.
`- (instancetype)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil;`

This is commented in the UIViewController.h documentation from `apple`.
`/*
  The designated initializer. If you subclass UIViewController, you must call the super implementation of this
  method, even if you aren't using a NIB.  (As a convenience, the default init method will do this for you,
  and specify nil for both of this methods arguments.) In the specified NIB, the File's Owner proxy should
  have its class set to your view controller subclass, with the view outlet connected to the main view. If you
  invoke this method with a nil nib name, then this class' -loadView method will attempt to load a NIB whose
  name is the same as your view controller's class. If no such NIB in fact exists then you must either call
  -setView: before -view is invoked, or override the -loadView method to set up your views programatically.
*/
- (instancetype)initWithNibName:(NSString *)nibNameOrNil bundle:(NSBundle *)nibBundleOrNil;`

So, basically, what I'm saying, is that we support init already. Just call init and the previous method will be invoked just fine.

**Comment by awd1559** [comment]
maybe make override init(nibName nibNameOrNil: String?, bundle nibBundleOrNil: NSBundle?) public in swift
